### PR TITLE
[samsungtv] documentation change codeowner

### PR DESCRIPTION
**Pull request by NickWaterton**
Update codeowner to @NickWaterton

### CODE REVIEW

**Comment by jlaur** [comment]
@paulianttila, would you no longer like to be a maintainer of this binding? Just to be sure.

**Comment by lsiepel** [comment]
> @paulianttila, would you no longer like to be a maintainer of this binding? Just to be sure.

FTR: i proposed to add @NickWaterton as second codeowner.

**Comment by jlaur** [comment]
> > @paulianttila, would you no longer like to be a maintainer of this binding? Just to be sure.
> 
> FTR: i proposed to add @NickWaterton as second codeowner.

Sure. I found these comments:
- https://github.com/openhab/openhab-addons/pull/11895#issuecomment-1407385266
- https://github.com/openhab/openhab-addons/pull/11895#issuecomment-1458894255

In the meantime, the binding was heavily rewritten during the last more than two years without any review comments/objections from @paulianttila, so I guess it would be logical to promote @NickWaterton as sole maintainer. It would still be nice with a confirmation from @paulianttila before doing so.

In any case - @kaikreuzer, can you add @NickWaterton to the contributor team?

**Comment by lsiepel** [comment]
Can you fix the conflict?

**Comment by lsiepel** [comment]
> Can you fix the conflict?

Ping @NickWaterton

**Comment by NickWaterton** [comment]
> > Can you fix the conflict?
> 
> Ping @NickWaterton

Sorry, didn't realize you were talking to me.

What are the conflicts? how would I resolve them? there is only one file edited.

**Comment by lsiepel** [comment]
> > > Can you fix the conflict?
> > 
> > 
> > Ping @NickWaterton
> 
> Sorry, didn't realize you were talking to me.
> 
> What are the conflicts? how would I resolve them? there is only one file edited.

If you update your fork main branch, and then your feature branch. You will get a notice about the conflict.

**Comment by NickWaterton** [comment]
> > > > Can you fix the conflict?
> > > 
> > > 
> > > Ping @NickWaterton
> > 
> > 
> > Sorry, didn't realize you were talking to me.
> > What are the conflicts? how would I resolve them? there is only one file edited.
> 
> If you update your fork main branch, and then your feature branch. You will get a notice about the conflict.

Ok,

I tried. I still don't know what the conflict was, or if this has resolved it.

File: CODEOWNERS
Comment (lsiepel) [inline CODEOWNERS:315]: The conflict was with salus, it was added to the exact same line. after you created the PR it was merged.
Now you fixed the conflict, but samsung is now above salus and it should be below (alphabetically) 
Sorry this small change is taking so much time.

Comment (NickWaterton) [reply CODEOWNERS]: Ok, moved it.

**Review by lsiepel — APPROVED** [review]
Thanks, LGTM

**Pull request merged**
